### ISSUE TITLE
Accessibility - 1010EZ: Focus style (yellow ring) does not meet contrast standards

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#56923]
**Issue description**

The design system's focus styles (yellow outline) don't meet WCAG 2.1 guidance for contrast for non-text. This is a known issue in the design system and work has been started. It effects all interactive elements: links, buttons, form fields, etc.

- [See this ticket with some suggestions on a path](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/654).
- And work on a solution, which has been paused for other priorities: [1313](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1313), [1314](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1314), [1315](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1314).

This is a placeholder ticket so we can pull in the design system updates when it's fixed there.

![Veteran information screen with address information. Yellow outline around country select field.](https://images.zenhubusercontent.com/628e7b9093a9366153cb221c/40c011e1-4e7a-4f4e-b2d8-46a6da80e817)

**Remediation guidance**

Implement better color contrast so it meets the 3:1 standard for non-text elements.

**Does this impact visual design?**

Yes.

**Relevant standards**

[1.4.11 Non-text contrast](https://www.w3.org/TR/WCAG21/#non-text-contrast)

**Comment by hdjustice** [comment]
Our team cannot work on this, it is dependent on the Design System team work.  Will keep an eye out for the changes to take affect and implement

**Comment by hdjustice** [comment]
Should be using form library, but should check for any imposter components being used that may still fall under this issue.
